com.unity.ml-agents.extensions/Runtime/Sensors/PoseExtractor.cs

```diff
@@ -77,60 +145,80 @@ public int GetParentIndex(int index)
             return m_ParentIndices[index];
         }
 
+        /// <summary>
+        /// Set whether the pose at the given index is enabled or disabled for observations.
+        /// </summary>
+        /// <param name="index"></param>
+        /// <param name="val"></param>
+        public void SetPoseEnabled(int index, bool val)
+        {
+            m_PoseEnabled[index] = val;
+        }
+
         /// <summary>
         /// Initialize with the mapping of parent indices.
         /// The 0th element is assumed to be -1, indicating that it's the root.
         /// </summary>
         /// <param name="parentIndices"></param>
-        protected void SetParentIndices(int[] parentIndices)
+        protected void Setup(int[] parentIndices)
```

Comment:
should the assumption be checked since this is a protected method on a public class?

Reply:
Done now.